ci-scripts/JenkinsFile-OAI-Container-GitHub

```diff
@@ -295,12 +295,11 @@ pipeline {
                     } catch (Exception e) {
                       echo 'OK if not present'
                     }
-                    // In case of the full build, we need the host RHEL8 credentials.
-                    if (params.REGRESSION_TEST) {
-                      sh('mkdir -p tmp/ca tmp/entitlement')
-                      sh('cp /etc/pki/entitlement/*pem tmp/entitlement')
-                      sh('sudo cp /etc/rhsm/ca/redhat-uep.pem tmp/ca')
-                    }
+                    // Copying the host RHEL8 credentials all the time
```

Comment:
NIT^2: is "all the time" referring to the previous state where this was only done for `REGRESSION_TEST`? The reader would not know this without going through the history - should it just be removed?